lib/jsmidgen.js

```diff
@@ -408,6 +408,15 @@ var Midi = {};
 		return bytes;
 	};
 
+	File.prototype.setTicks = function(num) {
+		// Do nothing in case of invalid input
+		if (!num || typeof num !== 'number') {
+			return;
+		}
+
+		this.HDR_SPEED = String.fromCharCode((num/256),  num%256);
```

Comment:
I'd prefer to keep the ticks per beat as an integer, and only serialize it to bytes just before we write it. In this case, that would mean removing `HDR_SPEED` entirely and storing in (say) `this.ticks` which defaults to 128.